src/DataSource.ts

```diff
@@ -55,6 +55,21 @@ export class DataSource extends DataSourceWithBackend<TimestreamQuery, Timestrea
     return query.rawQuery ?? '';
   }
 
+  private interpolateVariable = (value: string | string[]) => {
```

Comment:
nice, I'm guessing this is maybe something that can be shared in the sdk or maybe elsewhere if we are reusing this in multiple plugins?

Reply:
yes! I have a point in the design document to try to use a common class that cover this